libpermian/plugins/github/__init__.py

```diff
@@ -54,6 +54,10 @@ def __init__(self, *args, **kwargs):
             jinja_pkg_env = jinja2.Environment(loader=jinja2.PackageLoader('libpermian.plugins.github', 'templates'))
             self.template = jinja_pkg_env.get_template('output_text.j2')
 
+    @property
+    def fallbackSettingsSections(self):
+        return ['reportSender-'self.reporting.type, self.reporting.type, 'github', 'reportSenders']
```

Comment:
We are merging to devel branch, so I feel like this should be cleaned right now. Return just ['github', 'reportSenders'] and update the settings.